Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml

```diff
@@ -207,3 +207,13 @@
   cost: 2800
   category: Livestock
   group: market
+
+- type: cargoProduct
+  id: LivestockWhiteSpindle
+  icon:
+    sprite: Mobs/Animals/white_spider.rsi
+    state: white
+  product: CrateNPCWhiteSpindle
+  cost: 3900
```

Comment:
I'm not sure, but I think it's very expensive to produce silk, which is not very sought after yet.

Reply:
Ii is a bit expensive than a cow, but it is an exotic animal. Food is more necessary and these spiders can spawn webs anywhere and annoying everyone so.. for my opinion the price is normal.